public/src/firebase.ts

```diff
@@ -0,0 +1,122 @@
+import {
```

Comment:
I would move these "public/src" files out of "public" as well, for the same reasons as package.json.